homepage.html

```diff
@@ -307,10 +306,13 @@ <h3 style='text-align: center'>
                 search_result[index].date_end +
                 "&nbsp&nbsp&nbsp" +
                 "</td>";
-              var home_id_search = room_information[index].house_id;
-              var detail_search = "<a href=\"detail.html?home_id="+home_id_search+"\">details</a>";
+              var home_id_search = search_result[index].house_id;
+              var detail_search = "<a href=\"detail_.html?home_id="+home_id_search+"\">details</a>";  
```

Comment:
do you need to change **detail_** to **detail**?

Reply:
I need to change href to detail too and I forgot. I will fix it soon